nilearn/_utils/niimg_conversions.py

```diff
@@ -44,6 +92,10 @@ def check_niimg(niimg, atleast_4d=False):
         call nibabel.load on it. If it is an object, check if get_data()
         and get_affine() methods are present, raise TypeError otherwise.
 
+    ndim: integer, optional
+        Must be 3 or 4. Indicate the dimensionality of the expected niimg. An
```

Comment:
maybe put "Must be 3 or 4" in the docstring instead:

ndim: integer {3, 4}, optional